rclc/CMakeLists.txt

```diff
@@ -4,8 +4,28 @@ project(rclc)
 
```

Comment:
Please add a default C standard for the project like it's done in other packages, see https://github.com/ros2/rcl/blob/293aee7843131dcd4e85d606f6bd4d320d83255b/rcl/CMakeLists.txt#L15-L18 for an example